test/jdk/com/sun/jdi/TestScaffold.java

```diff
@@ -453,6 +453,10 @@ protected void traceln(String str) {
 
     protected void failure(String str) {
         println(str);
+        StackTraceElement[] trace = Thread.currentThread().getStackTrace();
+        for (StackTraceElement traceElement : trace) {
+            System.err.println("\tat " + traceElement);
+        }
```

Comment:
Thread.dumpStack() prints to System.errr.

Reply:
But it does so by creating an Exception and calling printStackTrace() on it. I didn't want that because it includes the Exception itself in the output, not just the stack trace.